src/lib/support/BUILD.gn

```diff
@@ -77,6 +77,7 @@ static_library("support") {
     "LifetimePersistedCounter.h",
     "PersistedCounter.cpp",
     "PersistedCounter.h",
+    "PersistentStorageUtils.h",
```

Comment:
Please use names that are not utils/helpers/support/misc.

Maybe `PersistenceDeviceOperationMacros.h`